### PR TITLE
Change /var/db to /var/lib on Linux deployments

### PR DESCRIPTION
**Pull request by 0ortmann**
###  :notebook_with_decorative_cover: Description

Change occurences of `/var/db/vast` to `/var/lib/vast` on Linux deployments.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚠️ The VAST database directory moved to `/var/lib/vast` for Linux deployments.
```
Comment (tobim) [inline CHANGELOG.md:14]:
```suggestion
- ⚠️ The default database directory moved to `/var/lib/vast` for Linux deployments.
```

**Review by tobim — CHANGES_REQUESTED** [review]

**Review by tobim — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚠️ The default database directory moved to `/var/lib/vast` for Linux deployments.
```
Comment (mavam) [inline CHANGELOG.md:14]: @dominiklohmann were you thinking to use the *breaking change* category for this?

Comment (dominiklohmann) [reply CHANGELOG.md:14]: No. It's not breaking for existing installations.

**Comment by 0ortmann** [comment]
I rebased on master and merged the CHANGELOG entries.

**Pull request merged**